### ISSUE TITLE
Update Rate Limit for getLivdData Azure Function

### ISSUE THREAD

**Issue by arnejduranovic** [CDCgov/prime-reportstream#16657]
# DevSecOps Issue

## Summary

`getLivdData` is an Azure Function with HTTPTrigger that provides the client a dump of our LIVD table. To enforce that this endpoint should not get called often because the data rarely changes, we should update the rate limit to 2 call per 5 minutes.

## Environment

- [ ] Local
- [ ] Dev
- [ ] Stage
- [ ] Prod

## Priority

- [ ] **Critical** - affecting prod systems
- [ ] **Major** - blocking major functionality, deployment, etc
- [ ] **Minor** - improvements, bug fixes
- [ ] **Nice-to-have** - feature request

### Blocks the following

- List any blockers
- Link any Github relevant issues

### Blocked by the following

- Link to any tickets that add context, etc

## Contact

**Team**:

**Slack Team Channel**:

**Slack Contact (Full Name):**

**Technical Team Lead:**

## Definition of Done

- [ ]

## Context Links

**Git Repo:**

**Relevant Code Links:**

**Azure**:

**Site URL**:

**Other**:

## Notes

-
